docs/usage_and_configuration/the_basics.md

```diff
@@ -200,6 +200,37 @@ project.
 "No". _Black_ is all about sensible defaults. Applying those defaults will have your
 code in compliance with many other _Black_ formatted projects.
 
+````{note}
+If you are using the linked configuration feature i.e. specifying `config` in the black config,
```

Comment:
You should first define the "linked configuration feature". Suggested wording:

```
You can use the `config = ` field to include configuration options from another configuration file.
Values set in the original configuration file have precedence over those in the included file.

Suppose `pyproject.toml` contains:
... (example below)
And `linked_config.toml` contains:
... (example below)
```

Also, no need for the `{note}`.